File: check_pass.php
<form method="POST">
    <input type="password" name="pwd" required />
    <input type="submit"/>
</form>
<?php
define('URL', 'https://api.pwnedpasswords.com/range/');
if(isset($_POST['pwd'])) {
    $sha1pass = strtoupper(sha1($_POST['pwd']));
    $headSha = substr($sha1pass, 0, 5);
    $tailSha = substr($sha1pass, 5, 39);
    $check = file_get_contents(URL.$headSha);
    $hashes = [];
    $lines = explode("\n", $check);
    foreach($lines as $line) {
        $hash = explode(':', $line);
        $hashes[$hash[0]] = $hash[1];
    }
    if(isset($hashes[$tailSha])) {
        echo 'Found occurences : '.$hashes[$tailSha];
    } else {
        echo 'Not found in haveibeenpwned.com database';
    }
    echo '<br />';
}
?>

File: check_pass.py
#!/bin/python3

import hashlib
import getpass
import requests

URL = 'https://api.pwnedpasswords.com/range/'

password_to_check = getpass.getpass(prompt=f'Enter password to check: ')

hash_sha1 = hashlib.new('sha1')
hash_sha1.update(password_to_check.encode('utf-8'))
password_hashed = hash_sha1.hexdigest().upper()

head_passord = password_hashed[0:5]
tail_passord = password_hashed[5:39]

r = requests.get(f'{URL}{head_passord}')
result = r.text

for line in result.splitlines():
    if tail_passord in line:
        print('Found occurences: ' + line.split(':')[1])
        exit()

print('Not found in haveibeenpwned.com database')
